cmd/incus-agent/dev_incus.go

```diff
@@ -66,6 +66,7 @@ var DevIncusConfigGet = devIncusHandler{"/1.0/config", func(d *Daemon, w http.Re
 		return smartResponse(err)
 	}
 
+	//nolint:revive //whyNoLint: the function is short and the shadowing is fine
```

Comment:
I'd still have made it `conf` or something instead just to avoid the shadowing should the function eventually get copy/pasted elsewhere or evolve to become problematic.